src/views/self/view_self.cpp

```diff
@@ -163,6 +167,26 @@ namespace big
 				ImGui::EndCombo();
 			}
 		});
+		ImGui::NewLine();
+		ImGui::Checkbox("Navigation", &g.self.nav);
+		components::options_modal("Navigation", [] {
```

Comment:
These should belong in the Vehicle menu, not self, as they are not relevant here.